src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -73,7 +74,12 @@ public Query buildSearchConceptsQuery(
     var domainClause = createDomainClause(concept, domainOption.getName());
 
     // c.concept_name, c.concept_id, COUNT(DISTINCT co.person_id) AS count
```

Comment:
so now this comment is slightly out of date

Reply:
Yep, this came from a merge, so I didn't know to update it. Maybe I should remove it?